### PR TITLE
New exciting story

### PR DESCRIPTION
**Pull request by alexsadliak**

### CODE REVIEW

**Comment by alexsadliak** [comment]
Please review this, @casey-collab

File: ukrainian/wait-more/out/try/index.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 Ви берете кілька листків с кущика, тягнете до рота та відчуваєте як ваше обличчя перекошує. 
-Яке ж воно кисле! Цікаво, воно все кисле, чи різні частини куща мають різні смаки.. Спробуємо?
\ No newline at end of file
+Яке ж воно кисле! Цікаво, воно все кисле, чи різні частини куща мають різні смаки.. Спробуємо?
+[Спробувати ще один листочок](parts/leaf/leaf.md)
+[Спробувати корінь](parts/root/root.md)
```
Comment (casey-collab) [inline ukrainian/wait-more/out/try/index.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: ukrainian/wait-more/out/try/index.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+[Спробувати гілку](parts/branch/branch.md)
```
Comment (casey-collab) [inline ukrainian/wait-more/out/try/index.md]: This link is also on the line directly after another link.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by alexsadliak** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**